product_barcode_generator/__init__.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 ###############################################################################
```

Comment:
Shorten headers like in https://github.com/OCA/maintainer-tools/blob/master/template/module/__init__.py